### PR TITLE
PDPDEVTOOL-5080 update readme for unit test

### PR DESCRIPTION
**Pull request by Montsant19**

### CODE REVIEW

File: packages/unit-testing/README.md [hunk 1]
```diff
@@ -12,17 +12,16 @@ Suitecloud Unit Testing allows you to use unit testing with [Jest](https://jestj
...
-  - **N/record**
-  - **N/search**
-- Allows you to create custom stubs for SuiteScript 2.X or custom modules.
+- Supports unit testing for SuiteScript 2.x files.
+- Provides SuiteScript 2.x stubs for SuiteScript modules.
+- Allows you to create custom stubs for SuiteScript 2.x or custom modules.
 - Supports code coverage capabilities through Jest.
 
+For more information about the available SuitScript 2.x modules, see [SuiteScript 2.x Modules](https://nlcorp.app.netsuite.com/app/help/helpcenter.nl?fid=chapter_4220488571.html).  
+For more information about all the mockable stubs, check the CORE_STUBS list in packages\unit-testing\jest-configuration\SuiteCloudJestConfiguration.js.
```
Comment (davidecorreu) [inline packages/unit-testing/README.md:21]: I think we could use a link to the file here like in the line above for SuiteScript 2.x Modules, naming it SuiteCloudJestConfiguration.js

File: packages/unit-testing/README.md [hunk 2]
```diff
@@ -60,18 +59,18 @@ However, if you want to configure SuiteCloud Unit Testing manually, do the follo
...
 8. From the root of your SuiteCloud project folder, run `npm test`
-to run your test. You should see an output similar to following:
+to run your test. You should see an output similar to the following:
     ```
     PASS  __tests__/sample-test.js
     Basic jest test with simple assert
         √ should assert stings are equal (2ms)
     ```
->💡 If you want to run tests with coverage, from the root of your SuiteCloud project folder, run `npm test --coverage`.
+>💡 If you want to run tests with coverage, run `npm test --coverage` from the root of your SuiteCloud project folder.
```
Comment (davidecorreu) [inline packages/unit-testing/README.md:68]: I'm not sure that we actually provide code coverage. I'll check it and let you know.

Comment (davidecorreu) [reply packages/unit-testing/README.md:68]: Confirmed, if we run `npm test --coverage` we have the same output as `npm test` , so there is no test coverage setup.

File: packages/unit-testing/README.md (hunk 1 above)
Comment (Montsant19) [reply packages/unit-testing/README.md:21]: OK, I somehow understood I had to point to the folder. Changed.

File: packages/unit-testing/README.md (hunk 2 above)
Comment (Montsant19) [reply packages/unit-testing/README.md:68]: Changed.

**Review by antoniofruci — APPROVED** [review]

File: packages/unit-testing/README.md [hunk 3]
```diff
@@ -12,17 +12,15 @@ Suitecloud Unit Testing allows you to use unit testing with [Jest](https://jestj
...
-  - **N/record**
-  - **N/search**
-- Allows you to create custom stubs for SuiteScript 2.X or custom modules.
-- Supports code coverage capabilities through Jest.
+- Supports unit testing for SuiteScript 2.x files.
+- Provides SuiteScript 2.x stubs for SuiteScript modules.
+- Allows you to create custom stubs for SuiteScript 2.x or custom modules.
+
+For more information about the available SuitScript 2.x modules, see [SuiteScript 2.x Modules](https://nlcorp.app.netsuite.com/app/help/helpcenter.nl?fid=chapter_4220488571.html).  
+For more information about all the mockable stubs, see the CORE_STUBS list in the [Stubs](https://github.com/oracle/netsuite-suitecloud-sdk/tree/master/packages/unit-testing/stubs) folder.
```
Comment (davidecorreu) [inline packages/unit-testing/README.md:20]: CORE_STUBS is an array that is defined inside `packages\unit-testing\jest-configuration\SuiteCloudJestConfiguration.js`, each element of the array includes the name we give to the mocked module, and the file it's referencing inside `packages/unit-testing/stubs`.
If we make reference to CORE_STUBS the link should be to [SuiteCloudJestConfiguration.js](https://github.com/oracle/netsuite-suitecloud-sdk/tree/masterpackages\unit-testing\jest-configuration\SuiteCloudJestConfiguration.js) file.

Comment (davidecorreu) [reply packages/unit-testing/README.md:20]: I also think that the link can be relative right: [SuiteCloudJestConfiguration.js](./jest-configuration\SuiteCloudJestConfiguration.js) 

```
as maybe the comment above is formatted by markdown,
here it is the same content inside a code block:
 [SuiteCloudJestConfiguration.js](./jest-configuration\SuiteCloudJestConfiguration.js) 
```

Comment (Montsant19) [reply packages/unit-testing/README.md]: Updated!

**Review by davidecorreu — APPROVED** [review]

**Pull request merged**